target/linux/ath79/image/generic.mk

```diff
@@ -3237,6 +3237,17 @@ define Device/xiaomi_mi-router-4q
 endef
 TARGET_DEVICES += xiaomi_mi-router-4q
 
+define Device/ditel_dt-acu100
+  SOC := qca9531
+  DEVICE_VENDOR := Ditel
+  DEVICE_MODEL := DT-ACU100
+  DEVICE_PACKAGES := -kmod-ath9k -uboot-envtools kmod-usb2
+  IMAGE_SIZE := 15936k
+  LOADER_FLASH_OFFS := 0x50000
+  SUPPORTED_DEVICES += dt-acu100
```

Comment:
Is this additional `SUPPORTED_DEVICES` necessary?

Reply:
Okay, i will remove it.